SimG4CMS/HcalTestBeam/interface/HcalTB06Analysis2015.h

```diff
@@ -0,0 +1,58 @@
+#ifndef SimG4CMS_HcalTestBeam_HcalTB06Analysis2015_H
```

Comment:
@civanch - we use code versioning systems instead of copying and old classes and updating them. Why do we need two versions of this?